lib/security-middleware-factory/index.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const _ = require('lodash');
+const helmet = require('koa-helmet');
+
+class SecurityMiddlewareFactory {
+
+  constructor(options) {
+    this._config =  _.merge({}, this.defaultConfig, options);
+  }
+
+  get defaultConfig() {
+    return {
+      csp: {
+        directives: {
+          defaultSrc: ["'self'"],
+          scriptSrc: ["'self'"],
+          styleSrc: ["'self'"],
+          imgSrc: ["'self'"],
+          frameAncestors: ["'self'"],
+          reportUri: 'about:blank',
+        },
+        reportOnly: true
+      },
+      hsts: {
+        maxAge: 30,
+        includeSubdomains: true,
+        preload: false
+      },
+      xssFilter: true,
+      noSniff: true
```

Comment:
I'd rename to `useNoSniff`, since this isn't the configuration of the xssFilter middleware.